pypit/arsave.py

```diff
@@ -126,22 +126,53 @@ def save_extraction(slf, sciext, scidx, scierr=None, filename="temp.fits", frame
 
 
 def save_master(slf, data, filename="temp.fits", frametype="<None>", ind=[],
-                extensions=None, keywds=None, names=None):
-    """ Write a MasterFrame
+                        extensions=None, keywds=None, names=None):
+    """ Wrapper to core_save_master
+    May be Deprecated
+
     Parameters
     ----------
     slf
     data
     filename
     frametype
     ind
+    extensions
+    keywds
+    names
+
+    Returns
+    -------
+
+    """
+    if len(ind) > 0:
+        raw_files=slf._fitsdict['filename']
+    else:
+        raw_files=None
+    core_save_master(data, filename=filename, frametype=frametype, ind=ind,
+                     extensions=extensions, keywds=keywds, names=names,
+                     raw_files=raw_files)
```

Comment:
Similar comment to above.  In the main code, can you call:

```
    core_save_master(data, filename=filename, frametype=frametype, ind=ind,
                     extensions=extensions, keywds=keywds, names=names,
                     raw_files=raw_files=slf._fitsdict['filename'] if len(ind) > 0 else None)
```
instead?

Reply:
This one is a bit more subtle but will disappear with a future
refactor nonetheless.